nautobot/extras/models/change_logging.py

```diff
@@ -96,6 +96,17 @@ class ObjectChange(BaseModel):
 
     class Meta:
         ordering = ["-time"]
+        get_latest_by = "time"
```

Comment:
This was just for convenience so that `ObjectChange.objects.earliest()/latest()` can be used.